test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -1935,13 +2046,15 @@ def test_create_prototype_baseline_building
 #      'daylighting_control',
 #      'light_occ_sensor',
 #      'infiltration',
-      'hvac_baseline',
+#      'hvac_baseline',
 #      'hvac_psz_split_from_mz',
+#      'plant_temp_reset_ctrl',
 #      'sat_ctrl',
 #      'number_of_boilers',
 #      'number_of_chillers',
 #      'number_of_cooling_towers',
 #      'hvac_sizing',
+      'preheat_coil_ctrl'
```

Comment:
Could you please reactive all tests? I don't think your changes should make other tests fail but for future development it'd be nice to have them all active.

Reply:
Activated all test cases.